pkg/preflights/host-preflight.yaml

```diff
@@ -834,3 +835,18 @@ spec:
           - pass:
               when: "a-subnet-is-available"
               message: Specified CIDR is available.
+    - sysctl:
+        checkName: Sysctl Kernel Parameters
+        outcomes:
+          - fail:
+              when: 'net.ipv4.conf.default.arp_filter > 0'
+              message: "ARP filtering is enabled by default for newly created interfaces on the host. Disable it by running 'sysctl net.ipv4.conf.default.arp_filter=0'."
+          - fail:
+              when: 'net.ipv4.conf.default.arp_ignore > 0'
+              message: "ARP ignore is enabled by default for newly created interfaces on the host. Disable it by running 'sysctl net.ipv4.conf.default.arp_ignore=0'."
+          - fail:
+              when: 'net.ipv4.conf.all.arp_filter > 0'
+              message: "ARP filtering is enabled for all interfaces on the host. Disable it by running 'sysctl net.ipv4.conf.all.arp_filter=0'."
+          - fail:
+              when: 'net.ipv4.conf.all.arp_ignore > 0'
+              message: "ARP ignore is enabled for all interfaces on the host. Disable it by running 'sysctl net.ipv4.conf.all.arp_ignore=0'."
```

Comment:
I think you want a pass condition here too.

As I understand Troubleshoot, you hit a matching `when` and drop out there. So if someone has filter and ignore set incorrectly, they would only get the message that comes first in this list. Is that correct? If so, can we make this so that all failure conditions would show? Not sure if that needs to be two analyzers or something else.

Reply:
You're right, I misunderstood `troubleshoot` in that aspect. Tested it out locally now and looks 👍

Reply:
The separate analyzers looks good. I still think you need a `pass` for each in addition to the fail. See the other analyzers in there, like [here](https://github.com/replicatedhq/embedded-cluster/blob/9db918c9d1014bf414da1edbd7abf28ffab64208/pkg/preflights/host-preflight.yaml#L831-L836).